### DISCUSSION TITLE
Release cadence

### DISCUSSION THREAD

**Discussion by djeedai** [bevyengine/bevy#2940] [General]
Hi-

Following my first experience with Bevy and the fact many of the bugs reported have already been fixed (see #2918), and even others found since then that I think are also fixed or about to be (set `is_visible=false` on parent doesn't hide children; #838, #2087, ...), I've been trying to consume the git version directly instead of the old 0.5.0 version, which is already 6 months old. Unfortunately, this is proving extremely challenging, even with a few dependencies: `bevy_kira_audio` and `bevy-inspector-egui` as must-have, `bevy_webgl2` for wasm, and `bevy_prototype_debug_lines` as utility (but I even tried to forfeit that one, without much luck). I have not found so far a combination of versions that work together, mostly due to dependency pinning, or the fact latest `bevy_kira_audio` doesn't build with latest `bevy`.

So, I wanted to ask about the release cadence. I see 0.5.0 is 6 months old. I also see a fair amount of activity in commits, which is a good thing, but means the gap from latest release is huge. I understand there are discussions and a focus toward 0.6.0; however it looks to me that this is still some time away. I see [a Roadmap in Projects](https://github.com/bevyengine/bevy/projects/1) but it's not clear if that's 0.6 or just a backlog. I also see that [the Bevy 0.6 milestone](https://github.com/bevyengine/bevy/milestone/3) is "only" 63% complete, and with 16 issues open left, doesn't look like it's coming next week.

Could this project consider smaller, less chunky releases more often? I appreciate this is a difficult balance, as resources are probably scarce like most projects, and each release takes time. But leaving a large gap in releases also means _I think_ that a portion of developers, those not willing or not able to go down consuming the latest directly from git, are left behind.

Thanks! :)

**Comment by Nilirad** [comment]
Hi,
I hope those answers will help you :)

> I see a Roadmap in Projects but it's not clear if that's 0.6 or just a backlog.

The Roadmap in Projects is not tied to a particular version, it just defines the general goals of Bevy for the foreseeable future.

> the Bevy 0.6 milestone is "only" 63% complete, and with 16 issues open left, doesn't look like it's coming next week.

Yeah, 0.6 won't come out next week. However, most of those issues and open PRs are still not closed/merged because of priority reasons: the huge rendering rework is blocking many important PRs that will be merged before 0.6.

> Could this project consider smaller, less chunky releases more often?

I totally support this, like probably (almost) everyone else. This problem has been addressed in the #2799 discussion. Feel free to give it a quick read and drop your opinions there.

**Comment by djeedai** [comment]
Thanks @Nilirad for the detailed answers! I went and [left a comment](https://github.com/bevyengine/bevy/discussions/2799#discussioncomment-1452883) on the relevant section of the other discussion. It's great to see this has support already!
